cypress/e2e/bucket/filter-bucket.cy.js

```diff
@@ -0,0 +1,104 @@
+import login from "../../utils/login";
+import get from "../../utils/get";
+import { createBucket } from "../../utils/bucket";
+import { createRound } from "../../utils/round";
+
+describe("Bucket filtering", () => {
+  beforeEach(login);
+
+  const now = Date.now();
+  const roundSlug = `round-${now}`;
+
+  before(() => {
+    login();
+    createRound(roundSlug);
+  });
+
+  it("filters buckets on based on statuses", () => {
+    // Only 1 cancelled bucket is required
+    let cancelled = false;
+    for (let i = 0; i <= 4; i++) {
+      createBucket(roundSlug, `Bucket ${now}`);
+
+      cy.wait(10000);
```

Comment:
we should avoid stuff like this. https://filiphric.com/waiting-in-cypress-and-how-to-avoid-it

but fine for now

Reply:
Sure, I'll replace waits